crates/lib/src/identity.rs

```diff
@@ -82,13 +88,13 @@ impl Identity {
 
 impl fmt::Display for Identity {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str(&hex::encode(self.__identity_bytes))
+        f.pad(&self.to_hex())
```

Comment:
Curious: why `pad`?

Reply:
Figured that I may as well; it means that `{:<.5}`-type formatting strings will work, and given that we have a fully formatted string that we pass to the formatter that there's no reason not to.